payment_methods/Paypal_Express/EEG_Paypal_Express.gateway.php

```diff
@@ -566,10 +562,12 @@ public function itemize_list(EEI_Payment $payment, EEI_Transaction $transaction,
     public function _ppExpress_request($request_params, $request_text, $payment)
     {
         $request_dtls = array(
-            'VERSION'   => '204.0',
-            'USER'      => urlencode($this->_api_username),
-            'PWD'       => urlencode($this->_api_password),
+            'VERSION' => '204.0',
+            'USER' => urlencode($this->_api_username),
+            'PWD' => urlencode($this->_api_password),
```

Comment:
looks like the spacing on the three lines above got messed up

Reply:
meuh, looks good to me. Personally I prefer to NOT try to align all the array assignments (because that way, often when you add something, you need to realign everything). Code formatter is ok with this, so I hope it's ok, I'm just going to roll with this

Reply:
yup, I understand, that is why I approved it